docs/internals/howto-release-django.txt

```diff
@@ -680,6 +680,27 @@ You're almost done! All that's left to do now is:
 #. If this was a security release, update :doc:`/releases/security` with
    details of the issues addressed.
 
+#. If this was an ``rc`` pre-release:
+
+   #. Update the source translation catalogs on the recently released stable
+      branch:
+
+      .. code-block:: shell
+
+          django-admin makemessages -l en
+          django-admin makemessages -l en --domain=djangojs -i views/templates/i18n_catalog.js
+
+      Avoid committing a change to the ``.po`` files without any new
+      translations (:commit:`example commit
+      <d2b1ec551567c208abfdd21b27ff6d08ae1a6371>`).
```

Comment:
I am not 100% sure the commands that are supposed to be run here. This is from reading the discussion in https://github.com/django/django/pull/17277.